### DISCUSSION TITLE
Switch between multiple saved users

### DISCUSSION THREAD

**Discussion by terrisgit** [community/community#3305] [Mobile] [answered]
The ability to save multiple user accounts and choose one without needing to logout and login

**Comment by Learning2Bcoder** [comment]
> The ability to save multiple user accounts and choose one without needing to logout and login

Why do you have multiple accounts is a better question perhaps?

**Reply by thesohelshaikh** [reply]
A work and a personal account.

**Reply by offero** [reply]
Same. Work and personal account.

**Comment by candyho** [comment]
Hey, are you looking to switch between your enterprise and personal accounts? We have just shipped a Beta for GitHub Enterprise Server 3.0 support on mobile recently which lets you switch between enterprise and personal accounts in-app. You can find more information about GitHub Enterprise Server 3.0 Support on mobile here - https://docs.github.com/en/enterprise-server@3.0/github/getting-started-with-github/github-for-mobile

**Reply by getaaron** [reply]
No, it's between multiple GitHub.com accounts (ie a free personal account and a Team account for work)

**Reply by thesohelshaikh** [reply]
Yep. Looking to add multiple personal accounts and not enterprise accounts.

**Reply by Karuji** [reply]
The enterprise support is great! But I have a separate (non-enterprise) account for work, and it would be really useful to have both accounts on the app!

Especially with this like MFA having multiple accounts in a single app like GMail does would be super useful 😁

**Comment by OneCyrus** [comment]
I have an github "enterprise managed user" and a regular one. it would be great if we could add both account types to the app. enterprise managed users are not able to collaborate in repositories outside of the enterprise. so i switch a lot between those two.

**Reply by mc-zact** [reply]
This would be really appreciated, as others here I have a company account and a personal one would be awesome to be able to do Fast User Switching

**Comment by ghost** [comment]
great idea.

**Comment by Wladefant** [comment]
Duplicate of #3337. Not good because that means the votes get divided up

**Comment by aisgbnok** [comment]
[It looks like they finally added this today for the mobile app.](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/)

***Note:** Since this is under the category **Mobile**, I'm only talking about the mobile app.*

**Comment by byrneh** [comment]
for browser access it is possible to be logged into.multiple accounts at the same time on a single device by using browser profiles and switching between these browser profiles

**Comment by eliperkins** [comment, marked as answer]
Hey @terrisgit and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
